app/component/map/map-with-tracking.cjsx

```diff
@@ -0,0 +1,98 @@
+React       = require 'react'
+timeUtils   = require '../../util/time-utils'
+cx          = require 'classnames'
+ComponentUsageExample = require '../documentation/component-usage-example'
+Example               = require '../documentation/example-data'
+Map                   = require './map.cjsx'
+ToggleMapTracking     = require '../navigation/toggle-map-tracking'
+MapTrackActions = require '../../action/map-track-actions'
+
+class MapWithTracking extends React.Component
+  @contextTypes:
+    getStore: React.PropTypes.func.isRequired
+    executeAction: React.PropTypes.func.isRequired
+
+  constructor: ->
+    super
+    #Check if we have a position already
+    locationState = @context.getStore('PositionStore').getLocationState()
+    if locationState.hasLocation
+      initialZoom = 16
+      initialLat = locationState.lat
+      initialLon = locationState.lon
+    else
+      initialZoom = 11
+      initialLat = 60.17332
+      initialLon = 24.94102
+
```

Comment:
This same thing is now in map.cjsx and here. We should:
- move this information to config file as we want map to centered differently depending on regional vs national
- avoid copy pasting